transport/matrix/node.py

```diff
@@ -101,6 +98,22 @@ def _http_retry_delay() -> Iterable[float]:
 
         self._message_handler: Optional[MessageHandler] = None
 
+    def enqueue_global_messages(self):
+        if self._prioritize_global_messages:
+            self._global_send_queue.join()
+
+    @property
+    def raiden_service(self) -> 'RaidenService':
+        return self._raiden_service
+
+    @property
+    def config(self) -> {}:
```

Comment:
why {} instead of dict?

Reply:
> With CPython 2.7, using dict() to create dictionaries takes up to 6 times longer and involves more memory allocation operations than the literal syntax. Use {} to create dictionaries, especially if you are pre-populating them, unless the literal syntax does not work for your case.

[source](https://stackoverflow.com/a/17098017)

Reply:
(in any case, we're just dealing with a type hint here, but i think `{}` is a preferable choice when it can be made in lieu of the slightly more complex `dict`)